packages/replay-next/components/inspector/KeyValueRenderer.tsx

```diff
@@ -132,6 +132,9 @@ export default function KeyValueRenderer({
       nameClass = styles.VerticalName;
     }
   }
+  if (protocolValue.isGetterValue) {
+    nameClass += " " + styles.GetterValue;
+  }
```

Comment:
nit: An alternate way we sometimes do this is with a data attribute (e.g. `data-getter-value`). The primary benefit of this is that the corresponding CSS selector (class + data attribute) is [more specific](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity), so it doesn't rely on the order of style names in the CSS file to set the correct color.